src/backend/base/langflow/components/jigsawstack/text_to_sql.py

```
@@ -0,0 +1,88 @@
+
+    outputs = [
+        Output(display_name="TextToSQL Results", name="text_to_sql_results", method="text_to_sql"),
+    ]
+
+    def text_to_sql(self) -> Data:
+        try:
+            from jigsawstack import JigsawStack
+        except ImportError as e:
+            raise ImportError(
+                "JigsawStack package not found"
+            ) from e
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)